win32/src/winapi/kernel32/sync.rs

```diff
@@ -1,30 +1,48 @@
 //! Synchronization.  Currently all no-ops as we don't support threads.
 
-use crate::{winapi::types::HANDLE, Machine};
+use crate::{winapi::types::HEVENT, Machine};
 
 const TRACE_CONTEXT: &'static str = "kernel32/misc";
 
+pub struct EventHandle {
```

Comment:
I guess in win32 terminology, the HEVENT is an integer that is an event handle, and it maps to an underlying "event object", which might be a better name here (or maybe plain "Event").

Reply:
Yeah, I agree. We should probably fix `ResourceHandle` and `FindHandle` as well 👍

Reply:
Seems like the windows documentation calls this an "Event Object", so let's go with that 👍